pkg/models/type.go

```diff
@@ -0,0 +1,15 @@
+package models
+
+type Type string
+
+const TypePersonal Type = "personal"
```

Comment:
Instead of declaring the constants separately, you can group them together to make the code more concise and readable:

```
const (
    TypePersonal Type = "personal"
    TypeTeam     Type = "team"
)
```